src/styles/blocks/card.scss

```diff
@@ -0,0 +1,110 @@
+.card {
+  box-sizing: border-box;
+  width: $card-width;
+  display: flex;
+  flex-direction: column;
+  background-color: $color-neutral;
+  padding: 16px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  justify-content: space-between;
+  transition: all, 300ms;
```

Comment:
Avoid to use `all` to `transition`, apply certain parameters you want to smoose.